### DISCUSSION TITLE
Importing the module in Angular gives me an error

### DISCUSSION THREAD

**Discussion by dilyandimitrov** [add2cal/add-to-calendar-button#332] [Q&A] [answered]
Hey
Following the instructions for how to use that in Angular (v8) and I receive the following error:

```
ERROR in ./node_modules/add-to-calendar-button/dist/module/index.js 966:21
Module parse failed: Unexpected token (966:21)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
|       focusEl.blur();
|     }
>   }
| }
| function atcb_close(host, keyboardTrigger = false) {
```

I installed the package with NPM, added the `CUSTOM_ELEMENTS_SCHEMA` in app.module and I have `import "add-to-calendar-button";`. 

It is probably an Angular thing and not particularly an issue with the package but I cannot understand what it is.

Best,
Dilyan

**Comment by jekuer** [comment, marked as answer]
Hi @dilyandimitrov ,

This is neither an Angular, nor an Add-to-Calendar issue. It is caused by webpack not being able to transpile the Add to Calendar code to your old Ecma Script version (which is a result of your old Angular version).
I have published a new version (2.1.2), which holds better backwards compatibility and hopefully works within your stack.

However, just a friendly advice: You should really update your project to the latest Angular version 15. Version 8 is now 4 years old, which is an extremely long period within Frontend development and no longer supported by a lot of stuff. Working with outdated frameworks is a big risk. In terms of security (to be fair, a smaller one), but more important in terms of being compatible to any third party element.

All the best,
Jens
